README.adoc

```diff
@@ -1333,6 +1333,10 @@ NOTE: The alternative syntax `when x: ...` has been removed as of Ruby 1.9.
 
 Do not use `when x; ...`. See the previous rule.
 
+=== Semicolon in `in` [[no-in-pattern-semicolons]]
+
+Do not use `in pattern; ...`. Use `in pattern then ...`.
```

Comment:
Can you give an example?

Reply:
I've updated this PR (and tweaked the description a bit).